### PR TITLE
DEVX-1212: Validate confluentinc/examples/rbac demo end-to-end with 5.4 image

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: security/rbac/delta_configs/connect-avro-distributed.properties.delta [hunk 1]
```diff
@@ -52,7 +52,6 @@ config.providers=secret
 config.providers.secret.class=io.confluent.connect.secretregistry.rbac.config.provider.InternalSecretConfigProvider
 config.providers.secret.param.master.encryption.key=password1234
 
-config.providers.secret.param.kafkastore.topic=_secrets
```
Comment (C0urante) [inline security/rbac/delta_configs/connect-avro-distributed.properties.delta:55]: Just double-checking--you've verified that removing this property from the config provider's configuration doesn't break things, right?

Comment (ybyzek) [reply security/rbac/delta_configs/connect-avro-distributed.properties.delta:55]: @C0urante correct.

Comment (C0urante) [reply security/rbac/delta_configs/connect-avro-distributed.properties.delta:55]: 👍

**Review by C0urante — APPROVED** [review]
The Secret Registry-related changes lgtm

File: security/rbac/scripts/enable-rbac-control-center.sh [hunk 2]
```diff
@@ -74,10 +73,6 @@ echo -e "\n# Grant principal User:$USER_CLIENT_C the DeveloperRead role to Subje
 echo "confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Subject:${TOPIC2_AVRO}-value --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID"
 confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Subject:${TOPIC2_AVRO}-value --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID
 
-echo -e "\n# Grant principal User:$USER_ADMIN_C3 the ClusterAdmin role to the Schema Registry cluster (workaround for MMA-4870)"
-echo "confluent iam rolebinding create --principal User:$USER_ADMIN_C3 --role ClusterAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID"
-confluent iam rolebinding create --principal User:$USER_ADMIN_C3 --role ClusterAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID
-
```
Comment (brianbushree) [inline security/rbac/scripts/enable-rbac-control-center.sh:80]: 👍 good catch

**Review by brianbushree — APPROVED** [review]
C3 changes LGTM

File: security/rbac/delta_configs/control-center-dev.properties.delta [hunk 3]
```diff
@@ -1,11 +1,6 @@
 confluent.controlcenter.rest.authentication.method=BEARER
-confluent.controlcenter.auth.bearer.public.key.path=/tmp/tokenPublicKey.pem
 confluent.controlcenter.streams.security.protocol=SASL_PLAINTEXT
+public.key.path=/tmp/tokenPublicKey.pem
```
Comment (srpanwar-confluent) [inline security/rbac/delta_configs/control-center-dev.properties.delta:3]: I can confirm that with this change non-docker setup works.

**Pull request merged**